tls/s2n_alerts.c

```diff
@@ -253,7 +253,8 @@ int s2n_queue_writer_close_alert_warning(struct s2n_connection *conn)
     alert[0] = S2N_TLS_ALERT_LEVEL_WARNING;
     alert[1] = S2N_TLS_ALERT_CLOSE_NOTIFY;
 
-    struct s2n_blob out = { .data = alert, .size = sizeof(alert) };
+    struct s2n_blob out = { 0 };
+    s2n_blob_init(&out, alert, sizeof(alert));
```

Comment:
s2n_blob_init returns a posix code. We should probably posix guard all of these calls.